teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

```diff
@@ -742,116 +754,196 @@ _When the orientation of the way search results are presented is changed, either
+| Global - Direct Deposit Account Type | 99 | Hit | User indicated `checking`, `savings`, or `none` to direct deposit account type on either the edu-1995 form or the edu-10203 form |
+| Global - Service Before 1978 | 100 | Hit | User indicated `yes` or `no` to serving before 1978 on the edu-1995 form  |
+| Global - Service Branches | 101 | Hit | User indicated branch served in such as `army` or `navy`on the edu-1995 form  |
+| Global - Service Start Dates | 102 | Hit | CD set to the date entered as the user's/vet's service start date on the edu-1995 form |
+| Global - Service End Dates | 103 | Hit | CD set to the date entered as the user's/vet's service end date on the edu-1995 form |
+| Education - Field of Study / Degree | 104 | Hit | User indicated field of study and length of time on the form 22-10203 form (for example, `sixMonthsOrLess` or `Computer Science`) |
+| Education - Benefits Remaining | 105 | Hit | User indicated length of time for benefits remaining on the form 22-10203 form. Options include: `sixMonthsOrLess`, `none`, or `moreThanSixMonths` |
+| Health Care VAOS - Express Care Reason | 106 | Hit | CD set to the express care reason a user selects while scheduling an appointment. Options include: `back pain`, joint or muscle pain or minor injury`, etc. Page to schedule an appointment: www.va.gov/health-care/schedule-view-va-appointments/appointments/new-express-care-request/additional-details/ |
+| Health Care VAOS - Express Number of Cards | 107 | Hit | **No data is being collected** |
+| Health Care VAOS - Upcoming Number of Cards | 108 | Hit | **No data is being collected** |
+| Facility Locator Search - Service Type | 109 | Hit | Service type (for example, `UrgentCare` or `Optometrist`) collected from a facility locator search on this page: www.va.gov/facilities/ |
+| Facility Locator Search - Facility Type | 110 | Hit | Facility type (for example, `health` or `pharmacy`) collected from a facility locator search on this page: www.va.gov/facilities/ |
+| Health Care VAOS - Returned Item Type | 111 | Hit | Returned item type (for example, `upcoming` or `video_home`) from the VAOS flow and collected on this page: www.va.gov/health-care/schedule-view-va-appointments/appointments/  |
+| Health Care VAOS - Returned Number of Items | 112 | Hit | CD is set to an integer matching the number of items of a search on this page:         <br>www.va.gov/health-care/schedule-view-va-appointments/appointments/ |
+| Global - Dependent Debt | 113 | User | CD is set to either `Without Dependent Debt` or `With Dependent Debt` depending on the user's selection on this page: www.va.gov/manage-va-debt |
+| Type Ahead Options Keyword Selected | 114 | Hit | User selects a type ahead option. The values for this CD are endless. |
+| Type Ahead Options List | 115 | Hit | User views type ahead options list |
+| Phased Out Product | 116 | Session | CD value is set to either `type ahead` or `chapter 36` |
```

Comment:
_For context, this is our current solution for dealing with phased roll outs (where teams first roll out their products to a percentage of the total population)_

```suggestion
| Phased Out Product | 116 | Session | Description or name of product with a phased roll out |
```